Makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: slides example
+
+define data
+package main
+
+import (
+    "fmt"
+    "log"
+    "net/http"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+    fmt.Fprintf(w, "Hi there, I love %s!", r.URL.Path[1:])
+}
+
+func main() {
+    http.HandleFunc("/", handler)
+    log.Fatal(http.ListenAndServe(":8080", nil))
+}
+endef
+export data
```

Comment:
what does this do?

Reply:
This generates an example HTTP server. The idea is that people can follow the slides at their own pace and have some handy commands to generate things for them.

So, for example, the slides would say: `Blah blah ... we need an HTTP server ... blah blah ... run 'make example' ... blah blah` and you have an example file that you did not need to copypasta, but you still needed to do some clickity-clacking on the keyboard to get it.

The big idea being - doing stuff on your own is more engaging that copypasting something from the internet

Reply:
But I go to port 8000 not 8080

Reply:
Port 8000 is for the slides and 8080 is for the example HTTP server

Reply:
wanna talk about it?
because I really don't get it.